db/seeds.rb

```diff
@@ -87,22 +80,25 @@ def david_made_this
 protein_ingredients.each do |ingredient|
   Ingredient.create(name: ingredient, category: 'protein')
 end
-
 grain_ingredients.each do |ingredient|
   Ingredient.create(name: ingredient, category: 'grain')
 end
-
 vegetable_ingredients.each do |ingredient|
   Ingredient.create(name: ingredient, category: 'vegetable')
 end
-
 dairy_ingredients.each do |ingredient|
   Ingredient.create(name: ingredient, category: 'dairy')
 end
 
 puts "Created #{Ingredient.count} ingredients"
 
 
+puts "creating challenges"
+
+Challenge.build!(miles, david, attributes = {accepted: true, expiration: Time.new(2020, 03, 05), voting_end: Time.new(2020, 03, 10), mode: 'creative' })
```

Comment:
not sure whether this syntax structure works - I think the following is preferred:

Challenge.build!(miles, david, accepted: true, expiration: Time.new(2020, 03, 05), mode: 'creative')
I also got rid of the voting_end because that variable is only ever initialized when the meals are both served.